internal/mage/sdk/nodejs.go

```diff
@@ -39,7 +43,26 @@ func (t NodeJS) Test(ctx context.Context) error {
 
 // Generate re-generates the SDK API
 func (t NodeJS) Generate(ctx context.Context) error {
-	panic("FIXME")
+	c, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stderr))
+	if err != nil {
+		return err
+	}
+	defer c.Close()
+
+	return util.WithDevEngine(ctx, c, func(ctx context.Context, c *dagger.Client) error {
+		generated, err := util.GoBase(c).
+			WithMountedFile("/usr/local/bin/cloak", util.DaggerBinary(c)).
```

Comment:
Should this be mounted? I was assuming that this file would be in the `engine` image, which is platform-specific. If we are mounting the `cloak` binary, I can imagine issues if it's built on e.g. macOS, but we are running it in Linux... WDYT @sipsma @aluzzardi?

Reply:
Also, after we generate the `client.ts`, should we build the Node.js version of the client?

Reply:
This cloak binary is built by the `util.DaggerBinary()` helper. It is built inside dagger as well. They should be on the same platform.
For the Node.js version of the client, I would defer to @slumbering .
For my POV, I would say it would be better for DX. But I don't know the amount of work needed, nor the way to package it, etc.
That should be part of the generation pipeline as well, down the line.

Reply:
OK